core/json/variables.examples.json

```diff
@@ -18,21 +18,22 @@
 		"text"			:	"<big>Like us on</big><br>Facebook!"
 	},
 	"system":{
-		"map_center_lat"	:	"50.844441",
-		"map_center_long"	:	"4.363557",
-		"zoom_level"		:	"13",
-		"GMaps_Key"		:	"#GMAPS_KEY#",
-		"timezone"		:	"Europe/Paris",
-		"max_pokemon"		:	251,
-		"pokeimg_suffix"	:	"_flat.png",
-		"recents_show_iv"	:	true,
-		"recents_filter"	:	true,
+		"map_center_lat"		:	"50.844441",
+		"map_center_long"		:	"4.363557",
+		"zoom_level"			:	"13",
+		"GMaps_Key"				:	"#GMAPS_KEY#",
+		"timezone"				:	"Europe/Paris",
+		"max_pokemon"			:	251,
+		"nest_pokemon"			:	"1, 4, 7, 21, 23, 25, 27, 29, 32, 37, 41, 43, 50, 56, 58, 60, 63, 66, 69, 74, 77, 79, 81, 84, 86, 90, 92, 95, 98, 100, 102, 104, 108, 109, 111, 114, 116, 120, 123, 124, 125, 126, 127, 133, 138, 140, 152, 155, 158, 161, 170, 183, 185, 187, 190, 191, 193, 194, 200, 202, 203, 204, 206, 207, 209, 211, 213, 215, 216, 218, 223, 226, 227, 228, 231, 234, 241",
```

Comment:
use an array instead (btw I removed 21 and 161).
Then we can use line breaks as well:

```json
[1, 4, 7, 21, 23, 25, 27, 29, 32, 37, 41, 43, 50, 56, 58, 60, 63, 66, 69, 74,
77, 79, 81, 84, 86, 90, 92, 95, 98, 100, 102, 104, 108, 109, 111, 114, 116,
120, 123, 124, 125, 126, 127, 133, 138, 140, 152, 155, 158, 161, 170, 183,
185, 187, 190, 191, 193, 194, 200, 202, 203, 204, 206, 207, 209, 211, 213, 
215, 216, 218, 223, 226, 227, 228, 231, 234, 241]
```